test/unit/core/testServiceJourney.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const expect = require('chai').expect;
+const rewire = require('rewire');
+const JourneyMap = rewire('app/core/JourneyMap');
+const iacJourney = require('app/journeys/iac');
+const initSteps = require('app/core/initSteps');
+const steps = initSteps([`${__dirname}/../../../app/steps/ui`]);
+const formdata = require('test/unit/core/testServiceData.json');
+const StartPage = steps.StartPage;
+const EndPage = steps.EndPage;
+const ShutterPage = steps.ShutterPage;
+
+describe('ServiceJourneyMap.js', () => {
+    const currentStep = {};
+    describe('stepList()', () => {
+        it('should return the journey step list', (done) => {
+            const journeyMap = new JourneyMap(iacJourney);
+            const stepList = journeyMap.stepList();
+            expect(stepList).to.deep.equal(iacJourney.stepList);
```

Comment:
Instead of testing with both expected and actual same could we change this to not null and dob step should not exist